getting_started_modules.md

```diff
@@ -0,0 +1,167 @@
+## Getting Started
+
+### Goal of This Guide
+
+> :warning: Read the section `Description - what & why` in [README.md](/README.md) to get a quick introduction to what this repo is.
+
+The template you found this README in is specifically built to make it as easy and quick as possible to make terraform modules, and then test them inside the [vagrant-hashistack](https://app.vagrantup.com/fredrikhgrelland/boxes/hashistack) box. 
```

Comment:
No sure that grammar is correct, just an offer (had feeling that it's written in yoda style)
```suggestion
The template you found in this README is specifically built to make it as easy and quick as possible to make terraform modules, and then test them inside the [vagrant-hashistack](https://app.vagrantup.com/fredrikhgrelland/boxes/hashistack) box. 
```

Reply:
I think the grammar is correct, but a little awakward. With your suggestion the template is inside the README, but the README is found inside the template. I'm talking about the repo-template here.

I'll rephrase it!